editors/sc-ide/widgets/multi_editor.cpp

```diff
@@ -1020,7 +1020,12 @@ int MultiEditor::addTab( Document * doc )
     if(tdoc->isModified())
         icon = mDocModifiedIcon;
 
-    tabIdx = mTabs->addTab( icon, doc->title() );
+    int currentIndex = mTabs->currentIndex();
+    if (currentIndex == -1) {
+      tabIdx = mTabs->addTab( icon, doc->title() );
```

Comment:
please use 4-space indentation for consistency with the rest of the file. otherwise, looks good